src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -214,15 +214,27 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
     private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberString, String responseNumberString) {
         StringBuilder tableHeaderFragmentHtml = new StringBuilder();
         String tableHeaderFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_HEADER_FRAGMENT;
-
-        for(int i = 0 ; i < numOfRubricChoices ; i++) {
-            String tableHeaderCell = 
-                    FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
-                            "${qnIndex}", questionNumberString,
-                            "${respIndex}", responseNumberString,
-                            "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", rubricChoices.get(i));
-            tableHeaderFragmentHtml.append(tableHeaderCell + Const.EOL);
+        
+        if(getChoiceAssginedWeight(rubricChoices) == null){
+            for(int i = 0 ; i < numOfRubricChoices ; i++) {
+                String tableHeaderCell = 
+                        FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
+                                "${qnIndex}", questionNumberString,
+                                "${respIndex}", responseNumberString,
+                                "${col}", Integer.toString(i),
+                                "${rubricChoiceValue}", (rubricChoices.get(i) + "(" + (numOfRubricChoices-i) + ")"));
```

Comment:
Add a space before the opening bracket for better readability (not code-wise, but usability wise, i.e `Choice (1)` should be better than `Choice(1)`).